### DISCUSSION TITLE
오픈채팅방 일정 등록/삭제는 어떻게 하나요?

### DISCUSSION THREAD

**Discussion by mschadev** [storycraft/node-kakao#328] [Q&A] [answered]
버전은 `4.2.1`쓰고 있습니다.  
제가 못찾는건지 [TalkOpenChannel](https://storycraft.github.io/node-kakao/classes/index.talkopenchannel.html)에도 없던데 관련 메서드가 있는지 궁금합니다.

**Comment by storycraft** [comment, marked as answer]
톡 게시판 웹 api가 구현되어 있지 않아 현재 NodeKakao 라이브러리 내에 채팅방 게시판 관련 메서드는 없습니다

**Reply by storycraft** [reply]
v3에 미완성된 api 구현체가 존재하니 직접 사용시 이곳을 참고하세요
https://github.com/storycraft/node-kakao/blob/v3/src/api/channel-board-client.ts
